frontend/test/metabase/scenarios/admin/databases/list.cy.spec.js

```diff
@@ -15,7 +15,7 @@ describe("scenarios > admin > databases > list", () => {
 
   it("should not let you see saved questions in the database list", () => {
```

Comment:
@nemanjaglumac Does this test need to exist? Imo it does not.

Reply:
I guess there was some weird bug back then when that test was created? o.O